Uplift/Models/Gym.swift

```diff
@@ -127,4 +139,72 @@ struct DailyGymHours {
             dayOfWeek = 0
         }
     }
+
+    init(gymHoursDataId: AllGymsQuery.Data.Gym.Facility.Time?) {
+        if let gymHoursData = gymHoursDataId {
+            dayOfWeek = gymHoursData.day
```

Comment:
i trust you'll reorganize these variables 🤡

Reply:
@chalo2000 do you mean alphabetize? if so, does it really make logical sense to have closeTime before openTime, and closeTime & openTime separated?